src/logic.service.ts

```diff
@@ -353,7 +353,7 @@ export const handlePacs002 = async (transaction: Pacs002): Promise<void> => {
     transaction,
     DataCache: dataCache,
     metaData: {
-      prcgTmDP: calculateDuration(startTime),
+      prcgTmTMS: calculateDuration(startTime),
```

Comment:
https://github.com/frmscoe/frms-coe-lib/blob/086b4f977f70cfcd12a06d4ee006181d7e6f1fad/src/interfaces/rule/RuleRequest.ts#L14C16-L14C16

I think we need to update this also. Thoughts?

Reply:
Well totally Let me revert it from this PR, so I can just ask for a ticket for those changes as it might need redeployment of every processor

Thanks for that catch though 💯